cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -52,8 +51,9 @@ namespace cob_cartesian_controller
 
     struct MoveCircStruct
     {
-        double x_center, y_center, z_center, roll_center, pitch_center, yaw_center;
-        double start_angle, end_angle, radius;
+        geometry_msgs::Pose pose_center;
+        double start_angle, end_angle;
```

Comment:
Based on the derived SI units the angle is given in [rad] (https://de.wikipedia.org/wiki/Internationales_Einheitensystem). 

Do the values for start_angle and end_angle have to be in [rad]?